woocommerce/woocommerce
# Fix E2E tests for WordPress 6.5

> [!WARNING]  
> This PR is blocked by https://github.com/woocommerce/woocommerce/pull/45789


### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:







### How to test the changes in this Pull Request:

This PR fixes E2E tests for WordPress 6.5. This PR:
- updates `@playwright/test` to `1.39.0` -> same version used in Gutenberg.
- updates `@wordpress/e2e-test-utils-playwright`.
- fix canvas usage -> https://github.com/WordPress/gutenberg/pull/54911.

Please, keep in mind that this PR fixes only the failure of E2E tests, but it doesn't improve/refactor any E2E tests. We aim to make these kinds of improvements with https://github.com/woocommerce/woocommerce/issues/45347.


My idea is to merge this PR a few days before WordPress 6.5 will be released.



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Ensure that Playwright E2E tests don't fail.




### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

Fix E2E tests on WordPress 6.5

</details>
